docs/customizing_deis/router_settings.rst

```diff
@@ -34,9 +34,10 @@ setting                                      description
 =======================================      ==================================================================================================================================================================================================================================================================================================================================
 /deis/builder/host                           host of the builder component (set by builder)
 /deis/builder/port                           port of the builder component (set by builder)
+/deis/config/\*/whitelist                    comma separated list of IPs (or CIDR) allowed to connect to the application containers (set by controller) Example: "0.0.0.0:some_optional_label,10.0.0.0/8"
 /deis/controller/host                        host of the controller component (set by controller)
 /deis/controller/port                        port of the controller component (set by controller)
-/deis/domains/*                              domain configuration for applications (set by controller)
+/deis/domains/\*                             domain configuration for applications (set by controller)
```

Comment:
@carmstrong I could remove this one but as soon as somebody adds a `foo/*/bar` it will be parsed as a comment.